iocage_lib/ioc_plugin.py

```diff
@@ -887,58 +905,23 @@ def __fetch_validate_plugin__(self, plugin, plugins):
             except ValueError:
                 exit()
         else:
-            # Quick list validation
             try:
-                plugin = [
-                    i for i, p in enumerate(plugins)
-
-                    if plugin.capitalize() in p or plugin in p
-                ]
-                try:
-                    plugin = plugins[int(plugin[0])]
-                except IndexError:
+                plugins[plugin]
```

Comment:
Why force an exception if you could just do `if plugin not in plugins:` ?

Reply:
Updated @william-gr